README.md

```diff
@@ -4,9 +4,16 @@ I offer no support or anything related to this script. May have more success if
 # Installation
 From PowerShell:
 
+```
 git clone https://github.com/ima9rd/psdirect-queue.git  
 cd psdirect-queue  
 python -m venv env  
 ./env/Scripts/activate  
 pip install -r requirements.txt  
 python app.py
+```
+
+If you encounter errors such as `No module named win32com.client, No module named win32, or No module named win32api`, you will need to install `pypiwin32`:

```

Comment:
pypiwin32 should be installed as a dependency for pyttsx3 once that is added to requirements.txt

Reply:
This is directly from the pyttsx3 [documentation](https://pypi.org/project/pyttsx3/), and since I don't have a windows machine there is no way for me to test if it gets auto installed. Given that it's in the documentation I'm assuming it's not auto installed as a dep

Reply:
It was installed when I put pyttsx3 in requirements.txt and used pip to install it. We can leave the verbiage in, just know that it probably won't be an issue in reality.